pkg/console/operator/sync_v400.go

```diff
@@ -311,7 +311,18 @@ func (co *consoleOperator) SyncConfigMap(
 		return nil, false, "FailedManagedConfig", mcErr
 	}
 
-	defaultConfigmap, _, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, managedConfig, infrastructureConfig, consoleRoute)
+	useDefaultRouterCA := true
+	// We are syncing the `router-ca` configmap from `openshift-config-managed` to `openshift-console`.
+	// `router-ca` is only published in `openshift-config-managed` if an operator-generated default certificate is used.
+	// It will not exist if all ingresscontrollers user admin-provided default certificates.
+	// If the `router-ca` configmap in `openshift-console` exist we should mount that to the console container,
+	// otherwise default to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`
+	_, rcaErr := co.configMapClient.ConfigMaps(api.OpenShiftConsoleNamespace).Get(api.RouterCAConfigMapName, metav1.GetOptions{})
+	if rcaErr != nil && apierrors.IsNotFound(rcaErr) {
```

Comment:
what about other errors? this only covers 404.

Reply:
is there anything else that would matter?

Reply:
was thinking about it and wasn't really sure if any other errors should matter in this context... saying that not really sure what should be the `reason` and `type` of the created condition, if there will be other error type... also if it makes sense to even  create one in this case...

Reply:
Maybe we should just add a log and call it good:

```golang
if rcaErr != nil {
  klog.V(4).Infof("router-ca GET error: %s",rcaErr)
}
```
Agree with you, probably doesn't matter, unless there is some funny state we aren't thinking about.  But a `4+` log level dumping some detail may be helpful.